repository_service_tuf_api/tasks.py

```diff
@@ -2,24 +2,78 @@
 #
 # SPDX-License-Identifier: MIT
 
-from typing import Any, Optional
+import enum
+from datetime import datetime
+from typing import Any, Optional, Union
 
 from pydantic import BaseModel, Field
 
 from repository_service_tuf_api import repository_metadata
 
 
+class TaskState(str, enum.Enum):
+    PENDING = "PENDING"
+    RECEIVED = "RECEIVED"
+    STARTED = "STARTED"
+    RUNNING = "RUNNING"
+    SUCCESS = "SUCCESS"
+    FAILURE = "FAILURE"
+    REVOKED = "REVOKED"
+    REJECTED = "REJECTED"
+    RETRY = "RETRY"
+    IGNORED = "IGNORED"
```

Comment:
Only for style. I think is more a task here as we don't use the Celery `states` to build dynamically the enum class.
I tried to do It.

Reply:
I am sorry, but I don't understand what you mean by "it's a task here".
It will be more work to do that? But why? Can't we just import `celery` and add the specific `states`?

Reply:
I don't see any direct benefit, but I added in my last commit.